src/components/App/App.js

```diff
@@ -21,6 +23,19 @@ class App extends Component {
 		if (!this.state.prediction) this.setState({prediction: {index, img}});
 	}
 
+	loadUsers() {
+		fetch(`https://slack.com/api/users.list?token=${token}`)
+			.then(res => res.json())
+			.then(data => {
+				this.setState({users: data.members});
+			})
+			.catch(err => err);
```

Comment:
Would we want to show an error if we couldn't fetch the users? If we set this in state we could show it in render, e.g.

```
.catch(fetchSlackUsersError => { this.setState({ fetchSlackUsersError }) })

// ...

render() {
  return (
    <div>
      ...
      {this.state.fetchSlackUserError && <Error>Failed to fetch slack users</Error>
    </div>
  )
}
```

Reply:
Got an 'Error undefined' message so made an ErrorMessage component for now